locales/en.json

```diff
@@ -1,16 +1,87 @@
-        "whatIsThisQ": "What is this?"
+        "header": {
+            "headline": "About Find A Doc, the database reducing vaccine waste."
+        },
+        "kofi": {
+            "headline": "Buy us coffee",
+            "text": {
+                "0": "We are running this database after our 9-5 jobs. Help us stay awake!",
+                "1": "Consider supporting us if you find this project helped you or someone in your network."
+            },
+            "action": "Donate a cuppa"
+        },
+        "team": {
+            "header": {
+                "headline": "Meet the Team"
+            },
+            "moreContributors": {
+                "headline": "Plus many more members who have lent us a helping hand!",
+                "action": {
+                    "text": "Find them on our {frontend} and {localization} Github contributors pages",
+                    "frontend": "front end",
+                    "localization": "localization"
+                }
+            },
+            "join": {
+                "headline": "Does Find a Doc’s mission spark your interests?",
+                "subheadline": "Join us over on GitHub to see how you can contribute!",
+                "action": "I want to join!"
+            }
+        },
+        "timeline": {
+            "items": {
+                "0": {
+                    "date": "2021.06.17",
+                    "dateHeadline": "From idea to health database",
+                    "text": {
+                        "0": "On that day, LaShawn Toyoda had enough of the lack of direction from officials. A few hours later, Find a Doc is born.",
+                        "1": "This database aims to better circulate the latest information on healthcare services in Japan."
+                    },
+                    "textHeadline": "Watch our daughter, I gotta make something."
+                },
+                "1": {
+                    "date": "Presently",
+                    "dateHeadline": "A team of volunteers",
+                    "text": {
+                        "0": "Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the immigrant community informed about clinics offering waiting lists for vaccinations to reduce waste.",
```

Comment:
`Find a Doc is community-driven with the support of a team of volunteers. Currently, the team is focused on keeping the international community of Japan informed about clinics offering waiting lists for vaccinations to reduce waste.`